.claude/settings.local.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Does this need to be part of the PR? Just checking if this is intentional or if "leaked" in.

Reply:
Nope. Will remove.